ReactWindows/ReactNative/Modules/Image/InMemoryImageCache.cs

```diff
@@ -0,0 +1,153 @@
+﻿using ReactNative.Bridge;
+using System;
+using System.Collections.Generic;
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Threading;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace ReactNative.Modules.Image
+{
+    class InMemoryImageCache : IImageCache
+    {
+        private readonly IDictionary<string, Impl> _cache =
```

Comment:
eventually, we'll need to implement a TTL for the in-memory BitmapImage instances.